.vscode/launch.json

```diff
@@ -38,7 +38,7 @@
       ],
       "env": {
         "NODE_ENV": "development",
-        "TEST_FILTER": "^file_comment$"
+        "TEST_FILTER": "^functions$"
```

Comment:
What does this change do?  I'm suspicious that a dot file has something important for us to check in...

Reply:
Result of an accidental `git commit -am ...`. 

If you don't know what this file does, we should sit down and see how your workflow looks. This is a launch description for VS Code that allows you to debug tsickle from within VS Code very conveniently. This line here is the line that picks the test to debug.